src/Google/Ads/GoogleAds/Lib/V14/GoogleAdsClientBuilder.php

```diff
@@ -415,6 +416,51 @@ public function validate()
                 'The gRPC channel credential can only be set when the transport is "grpc".'
             );
         }
+        // Check if the version of the grpc extension installed by Composer is greater than that of
+        // the extension installed by PECL, throw an exception to remind the user to upgrade.
+        $grpcPackageVersion = phpversion('grpc');
+        $grpcComposerVersion = $this->getGrpcComposerVersion();
+        if (
+            !empty($grpcComposerVersion)
```

Comment:
what if `$grpcPackageVersion` is `false` and `$grpcComposerVersion` has some value?

Reply:
Nice catch. I think another condition `!empty($grpcPackageVersion)` is missing.

Reply:
exactly, in my setup where I don't have the extension installed it throws an exception :(

Reply:
Sorry for your inconvenience. :( Will create a PR soon.